src/libstd/f32.rs

```diff
@@ -1414,6 +1414,8 @@ mod tests {
         assert!((-0.0f32).asinh().is_sign_negative()); // issue 63271
         assert_approx_eq!(2.0f32.asinh(), 1.443635475178810342493276740273105f32);
         assert_approx_eq!((-2.0f32).asinh(), -1.443635475178810342493276740273105f32);
+        // regression test for the catastrophic cancellation fixed in 72486
+        assert_approx_eq!((-67452098.07139316f64).asinh(), -67452098.07139316f64.asinh());
```

Comment:
This is for `f64`, not for `f32`. 3000 would be enough for `f32`,
```rust
assert_approx_eq!((-3000.0f32).asinh(), -8.699514775987968673236893537700647f32);
```

Reply:
Whoops, good catch, thanks.